includes/controllers/class-user.php

```diff
@@ -752,6 +760,66 @@ public function delete_user( $request ) {
 		return hp\rest_response( 204 );
 	}
 
+	/**
+	 * Resend user verification email.
+	 *
+	 * @param WP_REST_Request $request API request.
+	 * @return WP_Rest_Response
+	 */
+	public function resend_email_verify( $request ) {
+
+		// Check authentication.
+		if ( is_user_logged_in() ) {
+			return hp\rest_error( 403 );
+		}
+
+		// Check permissions.
+		if ( ! get_option( 'hp_user_verify_email' ) ) {
+			return hp\rest_error( 403 );
+		}
+
+		// Get user email.
+		$email = sanitize_email( $request->get_param( 'email' ) );
+
+		if ( ! $email ) {
+			return hp\rest_error( 403, esc_html__( 'Email is not found', 'hivepress' ) );
+		}
+
+		// Get user object.
+		$user_object = get_user_by( 'email', $email );
+
+		if ( ! $user_object || $user_object->hp_user_verified ) {
+			return hp\rest_error( 404, esc_html__( 'User is not found or user has been already verified', 'hivepress' ) );
```

Comment:
You can re-use one of the available strings, e.g. "No users found", so there will be no new strings in Loco and the translation % on WordPress.org will not be lower after the update.